<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>948</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>imex1711</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>81</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1261</width>
         <height>65</height>
        </rect>
       </property>
       <widget class="my_qlabel" name="label_cams">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>1</width>
          <height>51</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Mouse detect</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOn</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1013</width>
             <height>723</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="my_qlabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="mouseTracking">
               <bool>true</bool>
              </property>
              <property name="tabletTracking">
               <bool>false</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="text">
               <string>Mouse detect</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="my_qlabel" name="label4x4_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>500</width>
          <height>351</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Mouse detect</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="my_qlabel" name="label4x4_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>10</y>
          <width>500</width>
          <height>351</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Mouse detect</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="my_qlabel" name="label4x4_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>500</width>
          <height>351</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Mouse detect</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="my_qlabel" name="label4x4_4">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>370</y>
          <width>500</width>
          <height>351</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>Mouse detect</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="pozycjaMyszy">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>65</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>X = 0, Y = 0</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="get64">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>375</y>
         <width>78</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>64 x 64 pix</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="get32">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>400</y>
         <width>78</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>32 x 32 pix</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="get256">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>250</y>
         <width>90</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>256 x 128 pix</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="get128">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>300</y>
         <width>90</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>128 x 128 pix</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonVideo">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>29</y>
         <width>65</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Video</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonPic">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>29</y>
         <width>73</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Picture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>220</y>
         <width>55</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mode:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="start_b">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>430</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>strat</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stop_b">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>430</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back_b">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>463</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="next_b">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>463</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="back_b_2">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>496</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt; 5</string>
       </property>
      </widget>
      <widget class="QPushButton" name="next_b_2">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>496</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>5 &gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openSaveB">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>562</y>
         <width>175</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Otwórz folder zapisu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="openPhoto">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>595</y>
         <width>175</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Wczytaj folder z zdjęciami</string>
       </property>
      </widget>
      <widget class="QLabel" name="zdarzenieMyszy">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>Zdarzenie</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="configCam">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>628</y>
         <width>175</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Dodaj kamerę</string>
       </property>
      </widget>
      <widget class="QLabel" name="picture_size">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="picture_size_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="picture_size_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="get128_64">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>325</y>
         <width>90</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>128 x 64 pix</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotate_r">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>529</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>^---- Prawo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotate_l">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>529</y>
         <width>80</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Lewo ----^</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="get256_2">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>275</y>
         <width>90</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>128  x 256 pix</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="get128_65">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>350</y>
         <width>90</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>64 x 128 pix</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_qlabel</class>
   <extends>QLabel</extends>
   <header location="global">my_qlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
